transport-interop/versionsInput.json

```diff
@@ -288,8 +288,13 @@
       "webtransport",
       "webrtc-direct"
     ],
-    "secureChannels": ["tls", "noise"],
-    "muxers": ["yamux"]
+    "secureChannels": [
+      "tls",
+      "noise"
+    ],
+    "muxers": [
+      "yamux"
+    ]
```

Comment:
Can we avoid reformatting this file please?

Reply:
Sure, will revert it!